Packs/FeedTAXII/ReleaseNotes/1_0_9.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### TAXII 2 Feed
+- Deprecated **reset_fetch_command** command.
```

Comment:
you should add what should be used instead, from Cortex XSOAR version 6.2

Reply:
done